common.blocks/forum-form/forum-form.js

```diff
@@ -0,0 +1,35 @@
+modules.define('forum-form', ['i-bem__dom', 'jquery'],
+    function(provide, BEMDOM, $) {
+
+provide(BEMDOM.decl(this.name, {
+    onSetMod: {
+        js: {
+            inited: function() {
+                var textarea = this.findBlockInside('textarea'),
+                    plainTextButton = this.findBlocksInside('button')[0],
+                    previewButton = this.findBlocksInside('button')[1];
+                    
+                textarea.bindTo('keyup', function() {
+                    $('.textarea').val() ? previewButton.delMod('disabled') : previewButton.setMod('disabled');
```

Comment:
it's better to use API of `textarea` block and use the fact that setting a modifier to false is the same as deleating it:
```js
previewButton.setMod('disabled', !!textarea.getVal());
```